File: visual-model-app/src/main/java/org/visual/model/app/controller/MainLayoutController.java
package org.visual.model.app.controller;

import jakarta.inject.Singleton;
import javafx.fxml.FXML;
import javafx.fxml.Initializable;
import lombok.val;
import org.visual.model.ui.control.IconButton;
//import org.visual.model.ui.bar.SystemTitleBar;

import java.net.URL;
import java.util.ResourceBundle;

@Singleton
public class MainLayoutController implements Initializable {
    public IconButton test;
//    @FXML
//    public SystemTitleBar titleBar;

    @Override
    public void initialize(URL location, ResourceBundle resources) {
        System.err.println(test.getIcon());
    }
}

File: visual-model-app/src/main/kotlin/org/visual/model/app/controller/MainLayoutController.kt
package org.visual.model.app.controller

import jakarta.inject.Inject
import javafx.fxml.FXML
import javafx.fxml.Initializable
import org.github.gestalt.config.Gestalt
import org.visual.model.shared.KSlf4j
import org.visual.model.shared.KSlf4j.Companion.log
import org.visual.model.ui.bar.CommonTitleBar
import org.visual.model.ui.bar.SystemTitleBar
import java.lang.Thread.sleep
import java.net.URL
import java.util.*
import java.util.concurrent.Executors
import java.util.concurrent.TimeUnit


@KSlf4j
class MainLayoutController : Initializable {

    @FXML
    private lateinit var titleBar: SystemTitleBar
    override fun initialize(p0: URL?, p1: ResourceBundle?) {
    }
}
